### PR TITLE
Add support for idempotency key in create_organization method

### PR DESCRIPTION
**Pull request by awolfden**
Add support for idempotency key in create_organization method

### CODE REVIEW

File: workos/organizations.py [hunk 1]
```diff
@@ -88,14 +88,23 @@ def create_organization(self, organization):
                     within the Organization allow profiles that are outside of the Organization's
                     configured User Email Domains. (Optional)
                 organization[domains] (list) - List of domains that belong to the organization
+            create_organization_options (dict) - Options configuration for creating an organization. (Optional)
```
Comment (blairlunceford) [inline workos/organizations.py:91]: Can we model this more after the Audit Logs implementation? https://github.com/workos/workos-python/blob/main/workos/audit_logs.py#L23

Key differences:
- We can just pass an `idempotency_key` parameter with a default value of `None`
- Construct the header like:
```
 headers = {}
        if idempotency_key:
            headers["idempotency-key"] = idempotency_key
```
- pass those `headers` into the request method

File: tests/test_organizations.py [hunk 2]
```diff
@@ -1,9 +1,4 @@
-import json
```
Comment (blairlunceford) [inline tests/test_organizations.py:1]: Why are these imports deleted?

**Review by blairlunceford — CHANGES_REQUESTED** [review]

File: tests/test_organizations.py (hunk 2 above)
Comment (awolfden) [reply tests/test_organizations.py:1]: the json import was not being used so I removed it to clean up the file. If this is being used somewhere that you see I'm happy to add it back in.

File: workos/organizations.py (hunk 1 above)
Comment (awolfden) [reply workos/organizations.py]: Updated ✅

File: tests/test_organizations.py (hunk 2 above)
Comment (blairlunceford) [reply tests/test_organizations.py:1]: Sounds good 👍

File: workos/organizations.py [hunk 3]
```diff
@@ -88,14 +88,21 @@ def create_organization(self, organization):
                     within the Organization allow profiles that are outside of the Organization's
                     configured User Email Domains. (Optional)
                 organization[domains] (list) - List of domains that belong to the organization
+            idempotency_key (str) - Idempotency key for creating an organization. (Optional)
+                idempotency_key (str) - A custom key to ensure uniqueness when creating a new organization.
```
Comment (blairlunceford) [inline workos/organizations.py:92]: Don't think we need this line anymore

File: workos/organizations.py [hunk 4]
```diff
@@ -88,14 +88,21 @@ def create_organization(self, organization):
...
 
         Returns:
             dict: Created Organization response from WorkOS.
         """
+        headers = {"Idempotency-Key": None}
+        if idempotency_key:
+            headers["Idempotency-Key"] = idempotency_key
```
Comment (blairlunceford) [inline workos/organizations.py:99]: Should be: `headers["idempotency-key"] = idempotency_key`

**Review by blairlunceford — CHANGES_REQUESTED** [review]

**Comment by awolfden** [comment]
Hey @blairlunceford I think I covered everything that you flagged, but let me know if you see anything else that needs to get updated!

**Review by blairlunceford — APPROVED** [review]
Looks good 🎉

**Pull request merged**